### DISCUSSION TITLE
Norvi ESP32 industrial device from Sri Lanka

### DISCUSSION THREAD

**Discussion by Cloolalang** [arendst/Tasmota#14909] [Q&A] [answered]
Hi all, 
I am curious if anyone has tried using Tasmota on this device:

https://norvi.lk/industrial-esp32-norvi-iiot/

(I have no personal connection to the maker of this)

P

**Comment by s-hadinger** [comment, marked as answer]
Nice but not cheap. I see no reason why Tasmota wouldn't work on these devices
